### ISSUE TITLE
Accord VideoSourcePlayer do not render videos

### ISSUE THREAD

**Issue by Ibanez1408** [accord-net/framework#1484]
**What would you like to submit?** (put an 'x' inside the bracket that applies)

  - [x ] question
  - [ ] bug report
  - [ ] feature request

**Issue description**

I have an accord videoSourcePlayer control in my form. Why is it that it is not rendering the video that I select? Please see my code below:

```
// Open video file using DirectShow
        private void openVideoFileusingDirectShowToolStripMenuItem_Click(object sender, EventArgs e)
        {
            if (openFileDialog.ShowDialog() == DialogResult.OK)
            {
                // create video source
                FileVideoSource = new FileVideoSource(openFileDialog.FileName);

                // open it
                sourceInitialiization = true;
                OpenVideoSource(FileVideoSource);
            }
        }

// Open video source
        private void OpenVideoSource(IVideoSource source)
        {
            // set busy cursor
            this.Cursor = Cursors.WaitCursor;

            // close previous video source
            CloseVideoSource();

            // start new video source
            videoSourcePlayer.VideoSource = new AsyncVideoSource(source);
            videoSourcePlayer.Start();

            // reset statistics
            statIndex = statReady = 0;

            // start timers
            timer.Start();
            alarmTimer.Start();
            //alarmTimer1.Start();

            videoSource = source;

            this.Cursor = Cursors.Default;
        }
```
  I even downloaded the sample video project from accord framework but I still couldn't get it to play a video. What am I missing? I don't get any errors the videoSourcePlayer control just says that the video has stopped. I need help with this please. Thank you.
Note: If you would like to support the development for this feature or resolution of this bug, consider adding a bounty to it later in https://www.bountysource.com/teams/accord-net/issues

**Comment by alecjames** [comment]
Did you handle event NewFrameReceived?  If not, add an empty handler for this event.

Which sample did you test? I found that "Video Player" plays my video files OK.

The two cameras demo needs the NewFrameReceived event added to show the camera images

In one test app that I made, I saw no video because the accord DLLs were not in the output directory. I copied them from another working project and it worked fine - I don't know what I did or didn't do that caused the DLLs not to be there.

**Comment by Ibanez1408** [comment]
Yes, I did Sir. Actually, this program runs in my laptop perfectly. I can even play a .flv file. But if I transfer it to another machine, say, a desktop, the desktop doesn't play the same video files the laptop can play. The desktop doesn't play any files anymore.
